fuzzers/backtrace_baby_fuzzers/command_executor/src/main.rs

```diff
@@ -24,13 +24,21 @@ use libafl::{
     inputs::{HasTargetBytes, Input},
     monitors::SimpleMonitor,
     mutators::scheduled::{havoc_mutations, StdScheduledMutator},
-    observers::{get_asan_runtime_flags, AsanBacktraceObserver, StdMapObserver},
+    observers::{get_asan_runtime_flags, StdMapObserver},
     schedulers::QueueScheduler,
     stages::mutational::StdMutationalStage,
     state::StdState,
     Error,
 };
 
+cfg_if::cfg_if! {
```

Comment:
You won't need a crate here, you can do
```rust
#[cfg(feature = "casr")]
use libafl::observers::CasrAsanBacktraceObserver;
#[cfg(not(feature = "casr"))]
use libafl::observers::AsanBacktraceObserver;
```
or maybe even just always enable AsanBacktraceObserver, even for the casr feature - why not?